packages/test-commerce-sdk-react/app/components/_app-config/index.tsx

```diff
@@ -23,6 +29,7 @@ const AppConfig = (props: AppConfigProps): ReactElement => {
             proxy="http://localhost:3000/mobify/proxy/api"
             locale="en-US"
             currency="USD"
+            headers={headers}
```

Comment:
I was getting a typescript lint error here.. are you seeing the same?

Reply:
Not on my editor. What error are you seeing?